templates/emails/thankyou.hbs

```diff
@@ -16,43 +12,31 @@ Subject: Thank you for your {{currency order.totalAmount currency=order.currency
 
 {{#if interval}}
   {{#if firstPayment }}
-    <p>We would like to thank you for your pledge to give {{currency order.totalAmount currency=order.currency}} per {{interval}}.
-      Your first payment was received on {{moment order.createdAt}}. Your contribution helps us continue our work.</p>
+    <p>Thank you for contributing {{currency order.totalAmount currency=order.currency}} per {{interval}} to {{collective.name}}.
+      Your first payment was made on {{moment order.createdAt}}. Your support makes a difference!</p>
       {{> charge_date_notice}}
   {{else}}
-    <p>We would like to thank you for your continued support. Your latest {{interval}}ly contribution of {{currency order.totalAmount currency=order.currency}} was received on {{moment transaction.createdAt}}.
-      Your contribution helps us continue our work.</p>
+    <p>Thank you for your continued support! Your latest {{interval}}ly contribution of {{currency order.totalAmount currency=order.currency}} was received on {{moment transaction.createdAt}}.</p>
   {{/if}}
 {{else}}
 
-<p>We would like to thank you for your contribution of {{currency order.totalAmount currency=order.currency}} on {{moment order.createdAt}}.
-  Your gift will help us continue our work.</p>
+<p>Thank you for your contribution of {{currency order.totalAmount currency=order.currency}} on {{moment order.createdAt}}. Your support makes a difference!</p>
 {{/if}}
 
-<p>You can follow our future activities and how we will be spending the money on our open collective page:
-  <a href="https://opencollective.com/{{collective.slug}}">https://opencollective.com/{{collective.slug}}</a>.
-</p>
-
-<h2>Help us raise more money!</h2>
-<p>Share this URL: <a href="https://opencollective.com/{{collective.slug}}">https://opencollective.com/{{collective.slug}}</a>
+<p>See our activities and transparent budget on
+  <a href="our page">https://opencollective.com/{{collective.slug}}</a>, and share the link to help us spread the word!
 </p>
 
 {{#if transactionPdf}}
-<h2>📎 Attachments</h2>
-<ul>
-  <li>A PDF receipt of your contribution to {{collective.name}}</li>
-  {{#if platformTipPdf}}
-    <li>A PDF receipt of your contribution to Open Collective</li>
-  {{/if}}
-</ul>
+<h2>📎 Receipt attached</h2>
```

Comment:
This was clearer before, explaining what were the different receipts. I would rollback in its previous state.

Reply:
@znarf : I've reverted that section back to it's original state. 👍🏽